app/src/main/AndroidManifest.xml

```diff
@@ -1,6 +1,8 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="com.mapzen.erasermap">
+    package="com.mapzen.erasermap"
+    android:versionCode="2"
+    android:versionName="0.1">
```

Comment:
No need to declare version code and version name here. It gets injected by Gradle at build time.